lib/manager/components/deployment/CurrentDeploymentPanel.js

```diff
@@ -229,7 +229,8 @@ class DeployJobSummary extends Component<{
   }
 
   _onClickDownloadGraph = () => {
-    this.props.downloadBuildArtifact(this.props.deployment, 'Graph.obj', this._getJobId())
+    const g = this.props.deployment.tripPlannerVersion === 'OTP_2' ? 'g' : 'G'
+    this.props.downloadBuildArtifact(this.props.deployment, `${g}raph.obj`, this._getJobId())
```

Comment:
Creating a whole variable for the single letter feels a little bit much. Couldn't we just have a `filename` variable that is either Graph.obj or graph.obj?

Reply:
Yes but that would be more